pkg/golinters/goanalysis/runners.go

```diff
@@ -90,11 +90,18 @@ func buildIssues(diags []Diagnostic, linterNameBuilder func(diag *Diagnostic) st
 		diag := &diags[i]
 		linterName := linterNameBuilder(diag)
 
+		var fix string
+		for _, suggestedFix := range diag.SuggestedFixes {
+			for _, textEdit := range suggestedFix.TextEdits {
+				fix += string(textEdit.NewText)
+			}
+		}
+
 		var text string
 		if diag.Analyzer.Name == linterName {
 			text = diag.Message
 		} else {
-			text = fmt.Sprintf("%s: %s", diag.Analyzer.Name, diag.Message)
+			text = fmt.Sprintf("%s: %s\n%s", diag.Analyzer.Name, diag.Message, formatSuggestFix(fix))
```

Comment:
it will always print additional empty line in case of no suggestions.
it would be better if you change the code to 
```suggestion
			text = fmt.Sprintf("%s: %s%s", diag.Analyzer.Name, diag.Message, formatSuggestFix(fix))
```
and add additional `\n` in the `formatSuggestFix` function

Reply:
Fixed it.